src/qmxgraph/page/api.js

```diff
@@ -596,25 +654,26 @@ graphs.Api.prototype.getDecorationParentCellId = function getDecorationParentCel
  * @param {mxCell} cell The cell.
  * @returns {object} The bounds object.
  */
-graphs.Api.prototype._getBoundsFromMxCell = function _getBoundsFromMxCell (cell) {
+graphs.Api.prototype._getBoundsFromMxCell = function _getBoundsFromMxCell(cell) {
     "use strict";
 
     var geometry = cell.getGeometry();
     var parent_anchor_position = null;
-    var x = geometry.x, y = geometry.y;
+    var x = geometry.x,
+        y = geometry.y;
 
     if (geometry.relative) {
-        parent_anchor_position = {'x': x, 'y': y};
+        parent_anchor_position = { x: x, y: y };
         x = geometry.offset ? geometry.offset.x : 0;
         y = geometry.offset ? geometry.offset.y : 0;
     }
 
     return {
-        'x': x,
-        'y': y,
-        'width': geometry.width,
-        'height': geometry.height,
-        'parent_anchor_position': parent_anchor_position
+        x: x,
```

Comment:
I didn't know about this (`x` and `'x'` both work the same here).

Reply:
yes it does